docs/getting-started-kubernetes.rst

```diff
@@ -133,15 +136,6 @@ Install the Network Operator from the NVIDIA NGC chart using custom values:
 
    helm show values nvidia/network-operator --version |network-operator-version| > values.yaml
 
-.. code-block:: bash
```

Comment:
Please return this part which had been removed

Reply:
Reverted, and added a description before it

Reply:
Thanks